nuttx-configs/esc35-v1/include/board.h

```diff
@@ -121,8 +121,8 @@
 
 #define STM32_CFGR_USBPRE       0
 
-#define GPIO_USART2_RX         GPIO_USART2_RX_2
```

Comment:
@Inspirati Is this for your debug? If so and it is not pinned out on the real HW - it does not belong in the released code